### ISSUE TITLE
Third-party repositories

### ISSUE THREAD

**Issue by altteo** [gitspeak/community#17]
At the moment it seams that I could not work with any repository. So I still need to go to github site. But it will be very cool if I could monitor my comments and issues in third-party repositories. Like here in gitspeak repo.

By the way, it also means that at the moment casts are also available only to the team of a repo.

**Comment by altteo** [comment]
By the way we need such community for Imba (gitter is not very convinient)

**Comment by aalemayhu** [comment]
Yeah, it would be convenient to be able to create GitSpeak issues by GitHub users inside of GitSpeak. Currently it's not possible since you would need to be a contributor to the organization but having the repositories you star or follow listed would be nice.
